templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -706,10 +706,10 @@ Outputs:
   DBEndpointAddress:
     Description: The database connection string.
     Value: !GetAtt 'CrowdDCStack.Outputs.DBEndpointAddress'
-  DBEncryptionKey:
-    Condition: UseDatabaseEncryption
-    Description: The alias of the encryption key created for Amazon RDS.
-    Value: !GetAtt 'CrowdDCStack.Outputs.DBEncryptionKey'
+  EncryptionKey:
+    Condition: UseEncryption
+    Description: The alias of the encryption key created for Amazon RDS and EFS.
```

Comment:
Is this better:
> The alias of the encryption key. This key is only created and used when you enable database and filesystem encryption.

Reply:
I was specific because we don't use the key for EBS encryption which are the volumes that are attached to the cluster nodes. EBS is encrypted with standard aws managed key, meanwhile EFS and RDS are encrypted with this custom key.